data/json/monsters/zanimal_upgrade.json

```
@@ -93,6 +93,43 @@
     ],
     "upgrades": { "half_life": 12, "into": "mon_skeleton_hulk" }
   },
+  {
+    "id": "mon_zombie_pig_brute",
+    "type": "MONSTER",
+    "name": { "str": "skull pig" },
+    "copy-from": "mon_zombie_pig",
+    "description": "This formerly wild boar appears to have been a truly massive specimen in life.  Stretching eight to nine feet in length, the most arresting feature of this animal is that it appears the horn of it's tusks have spread entirely across the face leaving a skull like appearance with deepset eyeholes.",
```

```suggestion
    "description": "This former wild boar appears to have been a truly massive specimen in life.  Stretching eight to nine feet in length, the most arresting feature of this animal is that the enamel of its tusks has spread across its face leaving a skull-like appearance with deep-set eyeholes.",
```
Tusks are the canine teeth of the boar.